diagnostic.md

```diff
@@ -36,13 +45,22 @@ Now, we want to get a single donut from the server. Write the `curl` request
 you'd use to retrieve a single donut, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request GET "http://www.example.com/donuts/$donut"
 ```
 
 Write an AJAX request to retrieve a single donut from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let getDonut = (response, error, allDonuts) {
+  console.log('Start Request')
+  $.ajax({
+    method: 'GET',
+    url: 'http://www.example.com/donuts/' + donut,
```

Comment:
looks like you passed in `allDonuts` but you're trying to use the variable `donut`